src/commands/redis/list-keys.ts

```diff
@@ -0,0 +1,83 @@
+import { flags } from '@oclif/command'
+
+import { Kommand } from '../../common'
+import { kuzzleFlags } from '../../support/kuzzle'
+
+export default class RedisListKeys extends Kommand {
+  static description = 'Lists keys stored in Redis'
+
+  static flags = {
+    help: flags.help(),
+    remove: flags.boolean({
+      description: 'Remove matching keys',
```

Comment:
I really don't like the `--remove` argument. I'd really prefer a separate command like `kourou redis:delete-keys MATCH`